projects/common/src/tokens/network-information.ts

```diff
@@ -0,0 +1,9 @@
+import {inject, InjectionToken} from '@angular/core';
+import {NAVIGATOR} from './navigator';
+
+export const NETWORK_INFORMATION = new InjectionToken<NetworkInformation | undefined>(
```

Comment:
I believe even if you return `undefined` from factory, injected value would be `null`. Maybe explicitly convert it to `null` here?